introduction.md

```diff
@@ -1,2 +1,7 @@
 Hello I'm John
-JavaScript
\ No newline at end of file
+JavaScript
+
+* HTML
+* CSS
+* SQL
+* JSON
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?